server.js

```diff
@@ -1,7 +1,7 @@
 const chalk = require('chalk');
 const app = require('./app');
 
-app.listen(process.env.PORT || 3000, function () {
+app.listen(process.env.PORT || 3000, function listen() {
```

Comment:
Did you get a linter error that didn't allow for an anonymous function here?

Reply:
ESLint throws a warning, not an error, for anonymous functions.